lib/themes/dosomething/paraneue_dosomething/includes/helpers.inc

```diff
@@ -361,11 +361,27 @@ function paraneue_dosomething_get_themed_reportback_item($data, $rogue_rb_item =
     return NULL;
   }
 
-  if ($data->id && !dosomething_kudos_disable_reactions($data->campaign->id) && !$rogue_rb_item) {
-    $data->kudos = dosomething_kudos_get_kudos_data_for_fid($data->id);
-  }
+  if ($data->id && !dosomething_kudos_disable_reactions($data->campaign->id)) {
+      if ($rogue_rb_item) {
+        // Find the fid from the rogue reference table
+        // $data->id = db_query("SELECT rogue_rbs.fid FROM {dosomething_rogue_reportbacks} rogue_rbs WHERE rogue_event_id = :rogue_event_id", array(':rogue_event_id' => $data->event_id))->fetchField();
+
+        $total_rogue_kudos = $data->kudos['total'];
+        $liked_by_current_user = $data->kudos['data']['current_user']['reacted'] ? true : false;
+
+        // Let the front end know that Rogue is powering the gallery.
+        drupal_add_js(
+          [
+            'dsRogue' => [
+              'poweringGallery' => true,
+            ]
+          ],
+          'setting'
+        );
+      }
```

Comment:
@DFurnes had to pass this back in because we have still have to be able to tell if Rogue is powering the gallery on the front end! If it is, we need to append the page number from the response to the proxy to `/reportbacks` to then make the correct Request to Rogue based on the pagination.